### PR TITLE
Updated user's avatar in navbar

### LINKED ISSUE #346: To give an option of logout Button in navbar after successful login of a user.
## I explored this website and found a issue in navbar.

I login successfully with my G-mail account in website but then too navbar shows the option to sign in and sign up.
If i would get an option to log-out there that would be more clear to the user to logout successfully from this application.

### PR DESCRIPTION
**Pull request by ritik-vashist**
Fixes #346

Here is the images after change

Updated user photo with hi,{user} message in the navbar after login

### CODE REVIEW

**Comment by ritik-vashist** [comment]
@Abhishek-kumar09 Sir, Updated the file src/compoments/navbar/index.js in this PR which is asked by you in #350.
Kindly review the changes commited.
Thank You

File: src/components/NavBar/index.js [hunk 1]
```diff
@@ -103,18 +104,31 @@ function NavBar() {
...
+                  Hi, {user.displayName}
+                </Typography>
+              )}
             </Hidden>
           </div>
-          <Avatar
-            className={classes.avatar}
-            src="https://media-exp1.licdn.com/dms/image/C560BAQHSXxzxdL45FA/company-logo_200_200/0/1588676153700?e=2159024400&v=beta&t=PEiMBARAmeCUpRrAuJZWev-F_oCZac_OunmUXcLuY5U"
-          />
+          {((user === "Undefined") && (user === "null")) ? (
```
Comment (Abhishek-kumar09) [inline src/components/NavBar/index.js:122]: undefined is a js property, don't put it in quotes.
remove quotes from unefined and null.

**Review by Abhishek-kumar09 — CHANGES_REQUESTED** [review]
I can't see your change even after logging in.
Can you provide the screenshot or method to reproduce.
image

File: src/components/NavBar/index.js (hunk 1 above)
Comment (ritik-vashist) [reply src/components/NavBar/index.js:122]: @Abhishek-kumar09  Sir, i removed the quotes and created new PR in #355.
I request you to check it once.

**Comment by ritik-vashist** [comment]
Updated changes in #360

**Comment by Abhishek-kumar09** [comment]
you could have done git pull to fetch latest changes in this PR.

I am closing this issue, since #360 is descendant.

**Pull request closed without merging**